spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -3,7 +3,7 @@
 feature 'Changing authentication factor' do
   describe 'requires re-authenticating' do
     before do
-      sign_up_and_2fa
+      @user = sign_up_and_2fa
```

Comment:
I agree with 1, 2 and 3 but disagree with  4. 

I've been in many projects that use rspec and not `let`. The reason I prefer these code bases is that `let` introduces readability problems. I've linked to this post several times, but one more time won't hurt, right? https://robots.thoughtbot.com/lets-not

Anyway, I am Ok with keeping `let` but would also be happy to see it go.